frontend/src/static/js/api.js

```diff
@@ -59,7 +59,7 @@ window.api = {
           };
 
           const searchParams = [
-            repo.organization, repo.repoName,
+            repo.displayName,
             obj.displayName, author,
```

Comment:
if we use repo.displayName, do we still need obj.displayName?

Reply:
right right, let me go check

Reply:
yep, still need, on line 51 on (paiseh, dun know how link), obj.displayName is the displayName of the particular author, (eg Cin, all those).

in case there is confusion, author refers to their github username.

Reply:
ic... we need to clean up the variable names one of these days to make it clearer.